weblate/trans/models/translation.py

```diff
@@ -1242,6 +1243,7 @@ def merge_translations(
 
             state = STATE_TRANSLATED
             if add_fuzzy or set_fuzzy:
+                # TODO: decide fuzzy state?
```

Comment:
`needs rewriting` but unsure.

Reply:
I think it can stay as is. This is result of manual user action (selected in the upload form).